newsroom-lib/src/main/java/com/theguardian/newsroom/eventhandling/EventHandlingDelegate.kt

```diff
@@ -0,0 +1,7 @@
+package com.theguardian.newsroom.eventhandling
+
+import com.theguardian.newsroom.model.Event
+
+interface EventHandlingDelegate {
```

Comment:
Name suggestions: `Desk`, `Editor`, `Department`, or, if we don't want to commit to a metaphor just yet, I'd probably cut it down to `EventHandler`.

(To me at least "delegate" has a more specific meaning: a class which implements a certain interface on behalf of another class?)